1-js/02-first-steps/11-logical-operators/article.md

```diff
@@ -1,289 +1,290 @@
-- If all operands have been evaluated (i.e. all were `false`), returns the last operand.
+- Հաշվում է օպերանդների արժեքները ձախից աջ։
+- Ամեն օպերանդի արժեք փոխակերպում է տրամաբանական տիպի։ Եթե արդյունքը `true` է այն կանգ է առնում և վերադարձնում այդ օպերանդի օրիգինալ արժեքը։
+- Եթե բոլոր օպերանդները հաշվվում են, և բոլորի արժեքը `false` է լինում, ապա վերադարձնում է վերջին օպերանդի արժեքը։
 
-A value is returned in its original form, without the conversion.
+Արժեքը վերադարձվում է իր օրիգինալ վիճակում, առանց փոխարկման(conversion)։
 
-In other words, a chain of OR `||` returns the first truthy value or the last one if no truthy value is found.
+Այլ կերպ ասած, ԿԱՄ `||`֊երից կազմված շղթան վերադարձնում է առաջին ճշմարիտ արժեքը, կամ վերջինը, եթե չկան ճշմարիտ արժեքներ։
 
-For instance:
+Օրինակ․
 
 ```js run
-alert( 1 || 0 ); // 1 (1 is truthy)
+alert(1 || 0) // 1 (1֊ը ճշմարիտ է)
 
-alert( null || 1 ); // 1 (1 is the first truthy value)
-alert( null || 0 || 1 ); // 1 (the first truthy value)
+alert(null || 1) // 1 (1֊ը առաջին ճշմարիտ արժեքն է)
+alert(null || 0 || 1) // 1 (առաջին ճշմարիտ արժեքը)
 
-alert( undefined || null || 0 ); // 0 (all falsy, returns the last value)
+alert(undefined || null || 0) // 0 (քանի որ բոլորը սխալական(falsy) են, վերադարձնում է վերջինը)
 ```
 
-This leads to some interesting usage compared to a "pure, classical, boolean-only OR".
+Սա հնարավորություն է տալիս կիրառել օպերատորը "մաքուր, ստանդարտ, միայն տրամաբանական ԿԱՄ"֊ի շրջանակներից դուրս։
 
-1. **Getting the first truthy value from a list of variables or expressions.**
+1. **Առաջին ճշմարիտ արժեքի ստացումը փոփոխականների կամ արտահայտությունների ցուցակից։**
 
-    For instance, we have `firstName`, `lastName` and `nickName` variables, all optional (i.e. can be undefined or have falsy values).
+   Օրինակի, ունենք `firstName`, `lastName` և `nickName` փոփոխականները, բոլորը ըստ ցանկության կարող են ստանալ սխալական կամ `undefined` արժեքներ։
 
-    Let's use OR `||` to choose the one that has the data and show it (or `"Anonymous"` if nothing set):
+   Եկեք օգտագործենք ԿԱՄ `||`֊ը ընտրելու համար մեկը որը ունի արժեք(ճշմարիտ) և ցույց տանք այն (կամ `"Anonymous"` եթե այդպիսի արժեք չի գտնվել)։
 
-    ```js run
-    let firstName = "";
-    let lastName = "";
-    let nickName = "SuperCoder";
+   ```js run
+   let firstName = "";
+   let lastName = "";
+   let nickName = "SuperCoder";
 
-    *!*
-    alert( firstName || lastName || nickName || "Anonymous"); // SuperCoder
-    */!*
-    ```
+   *!*
+   alert( firstName || lastName || nickName || "Anonymous"); // SuperCoder
+   */!*
+   ```
 
-    If all variables were falsy, `"Anonymous"` would show up.
+   Եթե բոլոր փոփոխականները լինեին սխալական, ապա կտեսնեինք `"Anonymous"`։
 
-2. **Short-circuit evaluation.**
+2. **Կրճատ հաշվարկ (Short-circuit evaluation)։**
 
-    Another feature of OR `||` operator is the so-called "short-circuit" evaluation.
+   ԿԱՄ `||`֊ի մեկ այլ հատկությունը այսպես կոչված "կրճատ հաշվարկ"֊ի հատկությունն է։
```

Comment:
I can see you removed some white space at the beginning of this and next sentences up to the section `## && (AND)`. Please restore them.